.github/ISSUE_TEMPLATE/comp_extensions_enrichment.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
nit: I just updated this template a bit for `https://raw.githubusercontent.com/open-telemetry/opentelemetry-dotnet-contrib/main/.github/ISSUE_TEMPLATE/comp_exporter_onecollector.md` if you want to grab that one.

Reply:
Done